start-site/src/main/resources/application.yml

```diff
@@ -105,8 +105,15 @@ initializr:
         mappings:
           - compatibilityRange: "[2.0.0.RELEASE,2.0.x.BUILD-SNAPSHOT]"
             version: 2.0.3.RELEASE
-          - compatibilityRange: "2.1.0.RELEASE"
+          - compatibilityRange: "[2.1.0.RELEASE,2.1.10.RELEASE]"
             version: 2.1.4.RELEASE
+          - compatibilityRange: "[2.1.10.RELEASE,2.1.x.BUILD-SNAPSHOT]"
+            version: 3.0.1.RELEASE
```

Comment:
A switch from `2.1.x` to `3.0` in the same Spring Boot range looks odd to me. Can you please clarify?

Reply:
Our v3.0.1 depends on boot v2.1.10. So if the user was to explicitly choose boot v2.1.9, for instance, then our transitive dependency on v2.1.10 might clobber that. That was my thinking.

Reply:
What I was trying to say is that a switch to a major release when you switch to a maintenance release of Spring Boot is quite unusual. I'd rather stick 3.0 for Spring Boot 2.2 if possible and keep the current mapped maintenance release.

Thoughts?